speechbrain/inference/ASR.py

```diff
@@ -477,3 +482,390 @@ def transcribe_batch(self, wavs, wav_lens):
     def forward(self, wavs, wav_lens):
         """Runs full transcription - note: no gradients through decoding"""
         return self.transcribe_batch(wavs, wav_lens)
+
+
+@dataclass
+class ASRStreamingContext:
+    """Streaming metadata for a streaming transducer model. Initialized by
```

Comment:
Should we change the name to TransducerASRStreamingContext? ASR context sounds like anything can work, but then the documentation says it's transducer.